### ISSUE TITLE
Hosting a game in pre-release lobby NPE's on lobby watcher

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#6484]
### How can the problem be recreated?

- launch local lobby
- host game, observer it can NPE in 'InGameLobbyWatcherWrapper#executeConnectivityCheck'

### What is an expected fix?

- connectivity check should be executed successfully

### (Optional) Additional information

Unless someone volunteers, I'll take this one up. I think a good solution would be to remove the explicit connectivity check endpoint and bundle it with the 'post-game' endpoint. Post-game would need to return a composite object with the connectivity result.

**Comment by prastle** [comment]
Explain in English since I head no problems hosting ?

**Comment by DanVanAtta** [comment]
@prastle there might have been a race condition to trigger this. I was able to repro this with a local lobby consistently and did assume the problem would carry forward to prerelease. One key difference between the two environments (prerelease and local) is latency. Regardless this problem would be fixed with #6519 as we've changed the design of when/how the lobby verifies internet connectivity to hosts (which auto-shuts down the host if it is not available on internet).
